libs/screen/image.ts

```diff
@@ -273,8 +273,10 @@ namespace helpers {
     function _setScreenBrightness(brightness: number) { }
 
     export function setScreenBrightness(img: Image, b: number) {
-        _helpers_workaround.brightness = b
-        _setScreenBrightness(_helpers_workaround.brightness)
+        if (b >= 10 && b <= 100) {
```

Comment:
Can you do clamp(10,100,...) instead?

Reply:
I think I'll go through the math functions in a few and rename the parameters in a bit so they're nice and useable, as it's annoying to remember the order for clamp (I've seen helpers elsewhere with it set as ``clamp(min, num, max)`` as well)